src/main/scala/com/campudus/tableaux/arguments.scala

```diff
@@ -105,7 +111,8 @@ object ArgumentChecker {
             true
           case _ =>
             false
-        })
+      }
+    )
```

Comment:
Makes sense somehow - unsure if I like the `field =>` without a `{...}`-block and the resulting formatting here.